dm/dm/master/scheduler/scheduler.go

```diff
@@ -2604,12 +2498,12 @@ func (s *Scheduler) RemoveLoadTask(task string) error {
 // return (worker, source) that is used by transferWorkerAndSource, to try to resolve a paused load task that the source can't be bound to the worker which has its dump files.
```

Comment:
"return (worker, source) that is used by transferWorkerAndSource"

in new source-worker relation, for the case transferWorkerAndSource(lworker = w1, lsource = s1, rworker = "", rsource = s2), we can bind the s2 to w1 without unbinding (w1, s1)

Reply:
Yes, we can. To me unbinding (w1, s1) and tryBound for s1 again will make the source load more balanced so I keep this logic here.